### PR TITLE
fix(cd-delete): dont delete all custom dich covariates

### PR DESCRIPTION
**Pull request by alilahrime**
### Bug Fixes
deleting custom dichotomous covariate doesnt delete all of them anymore
*filter was returning empty array each time

### CODE REVIEW

File: src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const CustomDichotomousSelect = ({
                 width: 300,
               }}
               actions={[
-                <DeleteOutlined onClick={(d) => handleCDRemove(d.uuid)} key='delete' />,
+                <DeleteOutlined onClick={() => handleCDRemove(cd.uuid)} key='delete' />,
```
Comment (pieterlukasse) [inline src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx:87]: what is `cd`?

**Review by nmetokishlubsky — APPROVED** [review]

File: src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx (hunk 1 above)
Comment (alilahrime) [reply src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx:87]: an object in the [`selectedDichotomousCovariates`] array its mapping over

Comment (alilahrime) [reply src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx:87]: i could rename

Comment (pieterlukasse) [reply src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx:87]: it's not the name, I was just checking if it is not `undefined`

**Comment by pieterlukasse** [comment]
tested and it works ✅

**Pull request merged**